Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/AzureADIdentityProtection_test.py

```diff
@@ -0,0 +1,126 @@
+
+    requests_mock.get(f'{client._base_url}/{url_suffix}?$top=50', json=api_response)
+    result = command(client, limit=50, **kwargs)
+
+    expected_values = api_response.get('value')
+    actual_values = result.outputs.get(f'{OUTPUTS_PREFIX}.{context_path}(val.id === obj.id)')
+    assert actual_values == expected_values
+
+    expected_next_link = api_response.get('@odata.nextLink')
+    if expected_next_link:  # risky_users_history_list does not have next link
+        actual_next_url = result.outputs.get(f'{OUTPUTS_PREFIX}.NextLink(val.Description === "{context_path}")', {}) \
+            .get('URL')
+        assert actual_next_url == expected_next_link
+
+
+@pytest.mark.parametrize('method,expected_output,url_suffix,kwargs',
+                         ((azure_ad_identity_protection_risky_users_confirm_compromised_command,
+                           '✅ Confirmed successfully.',
+                           'riskyUsers/confirmCompromised',
+                           {'user_ids': [dummy_user_id]}
+                           ),
+                          (azure_ad_identity_protection_risky_users_dismiss_command,
+                           '✅ Dismissed successfully.',
+                           'riskyUsers/dismiss',
+                           {'user_ids': [dummy_user_id]}
+                           )
+                          )
+                         )
+def test_status_update_commands(client, requests_mock, method, expected_output, url_suffix, kwargs):
+    """
+    Given:
+        - AAD Client
+        - User name whose status we want to update
+
+    When:
+        - Calling a user-status-changing method (dismiss, confirm compromised)
+
+    Then:
+        - Verify API request sent as expected
+        - Verify command outputs
+    """
+
+    requests_mock.post(f'{client._base_url}/{url_suffix}', status_code=204)
+    result = method(client, **kwargs)
+    assert requests_mock.request_history[0].json() == {'userIds': [dummy_user_id]}
+    assert result == expected_output
+
+
+def test_parse_list():
+    """
+    Given
+        - A Microsoft Graph List response (collection of objects)
+    When
+        - calling parse_list()
+    Then
+        - Validate output parsing
+    """
+    with open('test_data/risk_detections_response.json') as f:
+        response = json.load(f)
+
+    human_readable_title = "Risks"
+    context_path = "Risks_path"
+
+    outputs = parse_list(response, human_readable_title=human_readable_title, context_path=context_path).outputs
+    assert len(outputs) == 2
+
+    values = outputs[f'AADIdentityProtection.{context_path}(val.id === obj.id)'][0]
+    assert len(values) == len(response['value'][0])  # all fields parsed
+
+    next_link_dict = outputs[f'AADIdentityProtection.NextLink(val.Description === "{context_path}")']
+    assert next_link_dict == {'Description': context_path,
+                              'URL': 'https://graph.microsoft.com/beta/riskDetections?$skiptoken=dummy_skip_token'}
```

Comment:
@DeanArbel does this suffice for `parse_list()` testing?

Reply:
@dorschw Please also add a case for an empty response (unless that's expected to be an error)